components/brave_rewards/resources/rewards_panel/components/publisher_card.tsx

```diff
@@ -0,0 +1,171 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { LocaleContext, formatMessage } from '../../shared/lib/locale_context'
+import { HostContext, useHostListener } from '../lib/host_context'
+import { MonthlyTipAction } from '../lib/interfaces'
+import { ToggleButton } from './toggle_button'
+import { MonthlyTipView } from './monthly_tip_view'
+import { NewTabLink } from '../../shared/components/new_tab_link'
+import { VerifiedIcon } from './icons/verified_icon'
+
+import * as styles from './publisher_card.style'
+
+const unverifiedLearnMoreURL = ''
```

Comment:
Should this be set to something or still TBD?

Reply:
Good catch - thanks!